### PR TITLE
progress bar take 2

### PR DESCRIPTION
**Pull request by wdwatkins**
Following @jiwalker-usgs 's comments, not messing with any other functions.  It will also tell you when the job has been accepted (or if it failed I think), and will start the progress bar when the job actually starts.

### CODE REVIEW

File: R/check.R [hunk 1]
```diff
@@ -68,7 +68,7 @@ setMethod(f = "check",signature(.Object = "geojob"), definition = function(.Obje
 		} else if (process$status == "Process Started") {
 		  process$percentCompleted <- as.character(xpathApply(root, "//wps:ProcessStarted", 
 		                                                      xmlGetAttr, "percentCompleted"))
-		}
+		}
```
Comment (jiwalker-usgs) [inline R/check.R]: Nothing to worry about, but watch out for whitespace sneaking in

File: DESCRIPTION [hunk 2]
```diff
@@ -38,7 +38,8 @@ Imports:
     httr (>= 1.0.0),
     curl,
     sp,
-    utils
+    utils,
+    progress
```
Comment (jread-usgs) [inline DESCRIPTION]: maybe just go ahead and specify a minimum version of `progress` as the one you are using?

File: R/waitUntilFinished.R [hunk 3]
```diff
@@ -35,9 +36,18 @@ setMethod(f = "wait",signature(.Object = "geojob", sleep.time = "missing"), defi
 #' @aliases wait
 setMethod(f = "wait",signature(.Object = "character", sleep.time = "numeric"), definition = function(.Object, sleep.time){
   running <- running(.Object, retry = TRUE)
+  pb <- progress_bar$new(total = 100, clear = FALSE)
+  currentStatus <- 'NA'
```
Comment (jiwalker-usgs) [inline R/waitUntilFinished.R]: maybe `NA_character_` instead of `'NA'`

**Comment by jiwalker-usgs** [comment]
Also, can you tell why travis is failing?

**Comment by wdwatkins** [comment]
I see lots of `Xpath error: undefined namespace prefix`, does that indicate flaky internet when it goes to the URL that defines the namespaces?

**Comment by jread-usgs** [comment]
all good?

**Comment by wdwatkins** [comment]
:shipit: , unless you want to keep this on a separate branch for now.

**Pull request merged**